### ISSUE TITLE
React query dev tools not showing data

### ISSUE THREAD

**Issue by alxhghs** [expo/dev-plugins#39]
I've had great luck with most of the dev-plugins, but can't seem to get the react-query one to work. Is the plugin currently working? 

I've created a minimal test repo, demonstrating how queries are not showing in the Chrome browser.

https://github.com/alxhghs/react-query-dev-tools-test/tree/main

<img width="2269" alt="image" src="https://github.com/expo/dev-plugins/assets/13875792/91b98442-b258-4713-889c-46f360e6340a">

**Comment by wooller** [comment]
This might not be the same for you but for me I had some pending chrome updates to install I was having issues with the network tab in the chrome debugger too. After installing the updates and re-launching chrome this plugin started to work and so did the network tab.

Word of warning, once this plugin started working. It drastically reduced the performance of our app when running a debug build, even when not launching the plugin window. It may be different in other apps but in the end we needed to remove it.

**Comment by jthoward64** [comment]
> This might not be the same for you but for me I had some pending chrome updates to install I was having issues with the network tab in the chrome debugger too. After installing the updates and re-launching chrome this plugin started to work and so did the network tab.
> 
> Word of warning, once this plugin started working. It drastically reduced the performance of our app when running a debug build, even when not launching the plugin window. It may be different in other apps but in the end we needed to remove it.

Not surprising, the plugin intercepts and serializes every single event. As a semi-proof of concept I don't there there was any performance work done on it

**Comment by AronBe** [comment]
➕ on the performance issues, but I think you don't have to remove it, just comment out the hook and enable when you really need it

`  //  useReactQueryDevTools(queryClient)`

**Comment by reriveram** [comment]
I'm still having the same issue @alxhghs, any luck on this ?

**Comment by svipas** [comment]
Yeah I can confirm that this isn't working with Expo 51 & React Native 0.74.5

**Comment by envisean** [comment]
Been trying to get this working with Expo 51 & RN 74 w/o any luck here either
